package.json

```diff
@@ -8,8 +8,8 @@
   "scripts": {
     "preinstall": "rm -rf ./public/uv",
     "postinstall": "yarn run uv-install && yarn run uv-config",
-    "uv-install": "cp -r ./node_modules/universalviewer/dist ./public/uv",
```

Comment:
the most recent change to this file removed the shx because it would not work reliably. Do we think something has changed that makes putting it back effective? I wonder if the real bug is an every other time thing or something

Reply:
It is an every other time thing. Does that indicate a different issue/solution?